### ISSUE TITLE
Vue 3 - "createElement is not a function"

### ISSUE THREAD

**Issue by Fanna1119** [amendx/vue-dndrop#26]
related to #18 
on a new vue 3 project installing `yarn add vue-dndrop@next` .

```js
import { Container, Draggable } from "vue-dndrop";

```

> vue-dndrop.esm.js:2892 Uncaught TypeError: createElement is not a function

**Comment by amendx** [comment]
Hi, @Fanna1119 , is your version up to *[1.2.1](https://www.npmjs.com/package/vue-dndrop/v/1.2.1)* ? 
Also, if it is, check out the libs .js file and see if it's according to [this commit](https://github.com/amendx/vue-dndrop/commit/f4b5610c24db8c26225c93682a6c72c4f54a3f50#diff-8a5a55861d9c55b1114b179f58340ee6e3fccbb41f365d1f66f8fa1a38f5899d)

**Comment by Fanna1119** [comment]
> Hi, @Fanna1119 , is your version up to _[1.2.1](https://www.npmjs.com/package/vue-dndrop/v/1.2.1)_ ? Also, if it is, check out the libs .js file and see if it's according to [this commit](https://github.com/amendx/vue-dndrop/commit/f4b5610c24db8c26225c93682a6c72c4f54a3f50#diff-8a5a55861d9c55b1114b179f58340ee6e3fccbb41f365d1f66f8fa1a38f5899d)

I can confirm that my version in my package.json is in fact 1.2.1.

however browsing through the `vue-dndrop.esm.js`
The code does not match the new commits

line 2923

```js
  render: function (createElement) {
    return wrapChild(createElement, this);
  },
```

**Comment by amendx** [comment]
@Fanna1119 You're right. Give me a minute to fix it.

**Comment by amendx** [comment]
@Fanna1119 you're right! Something got messed up on the rollup. It's fixed on version [1.2.2](https://www.npmjs.com/package/vue-dndrop/v/1.2.2) safe and sound. 

Try to update it and see if it persists.

**Comment by Fanna1119** [comment]
@amendx seems to be fixed! thank you for the quick response! And a huge thank for the awesome library!
